doc/getting_started/installation_win.rst

```diff
@@ -306,13 +308,15 @@ To build for the Intel |reg| Quark |trade| (x86-based) Arduino 101:
 .. zephyr-app-commands::
   :zephyr-app: samples/hello_world
   :board: arduino_101
+  :host-os: unix
```

Comment:
looks like a copy/paste issue here; this should be' win'

Reply:
good catch, fixed